### DISCUSSION TITLE
ps_checkout 404 not found iframe

### DISCUSSION THREAD

**Discussion by taoufikjabbari-jcd** [PrestaShop/PrestaShop#32021] [Q&A]
Hi,

 

I got this issue with ps_checkout 2.21 and also these errors in the console  : 

![image](https://user-images.githubusercontent.com/76155048/229499810-d454295e-44d0-4304-9fe6-5f0a16608dd2.png)
![image](https://user-images.githubusercontent.com/76155048/229499830-c130c24a-ac94-400a-abef-7d44949f875e.png)

Any ideas please ?

**Comment by kpodemski** [comment]
Hello @taoufikjabbari-jcd 

Discussions are part of the open-source project. If you have any issues with PrestaShop Checkout, you should contact the support of this module.
